Drivers/Inc/BatteryBalancing.h

```diff
@@ -0,0 +1,54 @@
+/** BatteryBalancing.h
+ *  functions for battery balancing.
+ */
+#ifndef BATTERY_BALANCING_H
+#define BATTERY_BALANCING_H
+
+
+#include "LTC6811.h"
+#include "Voltage.h"
+
+/**
+ * @brief   Finds lowest voltage amongst the 31 modules. Meant to be called only in Balancing_Balance()
+ * @param   None
+ * @return  the  
+ */
+static uint16_t Balancing_Init(void);
```

Comment:
This seems like a misleading name for this function. The name `Balancing_Init()` seems like it should initialize the battery balancing hardware. Also, why is this in the header file if it is a helper function that is only called in `Balancing_Balance()` ?